pkg/runtime/runtime.go

```diff
@@ -140,6 +141,10 @@ func (a *DaprRuntime) Run(opts ...Option) error {
 	d := time.Since(start).Seconds() * 1000
 	log.Infof("dapr initialized. Status: Running. Init Elapsed %vms", d)
 
+	if a.daprHTTPAPI != nil {
```

Comment:
&& with a check for gRPCserver as well

Reply:
> && with a check for gRPCserver as well

gRPC server condition is taken care of in InitRuntime, if it is failed to start then error is logged as Fatal . Ready flag is set when init is done.

Reply:
> > && with a check for gRPCserver as well
> 
> gRPC server condition is taken care of in InitRuntime, if it is failed to start then error is logged as Fatal . Ready flag is set when init is done.

Great, could you please add a comment for this here.

Reply:
I have added a comment now.